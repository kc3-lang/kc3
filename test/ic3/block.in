quote do
  1
end
do
  1
end
quote do
  1
  2
end
do
  1
  2
end
quote do
  1
  2
  3
end
do
  1
  2
  3
end
quote do 1; end
do 1; end
quote do 1; 2; end
do 1; 2; end
quote do 1; 2; 3; end
do 1; 2; 3; end
