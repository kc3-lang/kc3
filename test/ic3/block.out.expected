do
  1
end
1
do
  1
  2
end
2
do
  1
  2
  3
end
3
do
  1
end
1
do
  1
  2
end
2
do
  1
  2
  3
end
3
